eo-parser/src/main/resources/org/eolang/parser/warnings/sparse-decoration.xsl

```diff
@@ -38,7 +38,7 @@ SOFTWARE.
           <xsl:attribute name="severity">
             <xsl:text>warning</xsl:text>
           </xsl:attribute>
-          <xsl:text>Sparse objects are prohibited</xsl:text>
+          <xsl:text>Sparse decoration is not prohibited</xsl:text>
```

Comment:
@maxonfjvipon the `"not"` is not supposed to be here, I believe?

Reply:
@yegor256 yes, you're right, my mistake